velox/dwio/common/tests/utils/DataSetBuilder.cpp

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/dwio/common/tests/utils/DataSetBuilder.h"
+
+#include "velox/dwio/common/tests/utils/BatchMaker.h"
+#include "velox/dwio/type/fbhive/HiveTypeParser.h"
+#include "velox/vector/FlatVector.h"
+
+namespace facebook::velox::test {
```

Comment:
`facebook::velox::dwio::test`?
The namespacing is already quite messy in dwio though, and it's hard to fix because there is quite a lot dependencies.

Reply:
@Yuhta I'm ok changing it to `facebook::velox::dwio::test`, but I don't see this namespace used anywhere in Velox. The files in the same folder were either in `facebook::velox::test` or `facebook::velox::dwio::common`. Are you sure we want to put DataSetBuilder itself to this new namespace? I created https://github.com/facebookincubator/velox/issues/2922 for this. How about we still use `facebook::velox::test` and address the namespace issue in a separate PR?

Reply:
Sure we can leave it as it is for now